D/datamash/build_tarballs.jl

```diff
@@ -0,0 +1,36 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "datamash"
+version = v"1.7.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://ftp.gnu.org/gnu/datamash/datamash-1.7.tar.gz", "574a592bb90c5ae702ffaed1b59498d5e3e7466a8abf8530c5f2f3f11fa4adb3")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/datamash-*/
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make -j${nproc}
+make install
+install_license COPYING
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
```

Comment:
I'd suggest to disable Windows here.  File [`src/decorate.c`](https://git.savannah.gnu.org/gitweb/?p=datamash.git;a=blob;f=src/decorate.c;h=1e1deff960e6245f4c4cb72b16a7c0628a66b253;hb=HEAD#l21) wants to unconditionally include the header file `sys/wait.h`, which however [isn't available for MinGW](https://www.gnu.org/software/gnulib/manual/html_node/sys_002fwait_002eh.html).  The weird thing is that [`HACKING.md`](https://git.savannah.gnu.org/gitweb/?p=datamash.git;a=blob;f=HACKING.md;h=f2cb8fad47a455628657d1f76c320a382c83392b;hb=HEAD#l225) mentions support for MinGW, but I don't see how this can work.  Unless `sys/wait.h` doesn't require extra MinGW packages that we don't have nor are mentioned in the documentation of datamash.

Reply:
There are two exe files generated. I think only decorate fails. How about disable decorate on Windows?

Reply:
The builder is your, so you are free to disable all what you want :slightly_smiling_face:   I only try to help to build as much as possible :wink:

Reply:
Thanks! I only use Ubuntu, so,